src/main/java/tsunami/security/scanner/utilities/KubeJavaClientUtil.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tsunami.security.scanner.utilities;
+
+import com.google.common.collect.ImmutableList;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.apis.AppsV1Api;
+import io.kubernetes.client.openapi.apis.CoreV1Api;
+import io.kubernetes.client.openapi.models.V1Deployment;
+import io.kubernetes.client.openapi.models.V1PersistentVolumeClaim;
+import io.kubernetes.client.openapi.models.V1Service;
+import io.kubernetes.client.util.Yaml;
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class KubeJavaClientUtil {
+  private static CoreV1Api coreV1Api = new CoreV1Api();
+  private static AppsV1Api appsV1Api = new AppsV1Api();
+
+  private interface Handler {
+    void handle(Object o) throws ApiException;
+  }
+
+  /* Use the map to relate a Class object to a Handler */
+  private static final Map<Class, Handler> apiCallByClass = new HashMap<Class, Handler>();
```

Comment:
Prefer using `ImmutableMap` and create it using `ImmutableMap.of()` method. This way you don't need the `static` block on line 45.